docs/en/flink/configuration/sink-plugins/Jdbc.md

```diff
@@ -22,7 +22,9 @@ Write data through jdbc
 
 ### driver [string]
 
-Driver name, such as `com.mysql.jdbc.Driver`
+Driver name, such as `com.mysql.cj.jdbc.Driver`.
+
+Warn: for license compliance, you have to provide MySQL JDBC driver yourself.
```

Comment:
It might be better to describe how to add `mysql-connector-java`.

Reply:
Good advise, I'll add more guiding info.